tox.ini

```diff
@@ -11,9 +11,9 @@ envlist =
 passenv = DD_TEST_CLIENT*
 usedevelop = true
 deps =
-    !integration: mock
```

Comment:
I think if we move the user agent test to tests/unit instead of tests/integration we wouldn't need to change deps.

Reply:
Yeah, but the whole unittest suite as it is set up now replaces the `RequestClient._session`, so we'd have to create a completely different setup just because of this one specific test. I don't mind doing that, but this is just simpler (plus it tests the real thing).